critiquebrainz/frontend/external/musicbrainz_db/place.py

```diff
@@ -0,0 +1,50 @@
+from mbdata import models
+from mbdata.utils import get_something_by_gid
+from critiquebrainz.frontend.external.musicbrainz_db import mb_session
+from critiquebrainz.frontend.external.musicbrainz_db.includes import check_includes
+import critiquebrainz.frontend.external.musicbrainz_db.exceptions as mb_exceptions
+from critiquebrainz.frontend.external.musicbrainz_db.serialize import to_dict_places
+from critiquebrainz.frontend.external.musicbrainz_db.helpers import entity_relation_helper
+from brainzutils import cache
+
+
+DEFAULT_CACHE_EXPIRATION = 12 * 60 * 60 # seconds (12 hours)
+THREAD_POOL_PROCESSES = 10
+
+
+def get_place_by_id(id):
```

Comment:
It's better to be more specific here and use `mbid` instead of `id`. I usually try to avoid using `id` since it's a built in function.